source/development/makefile.rst

```diff
@@ -0,0 +1,225 @@
+.. Copyright (C) 2018 Wazuh, Inc.
+
+.. _wazuh_makefile:
+
+Makefile options
+================
+
+This section contains instructions to customize the installation from Wazuh by compiling the source code before executing the installation script.
+
+You can also find here the different settings available for the ``Makefile``. Each setting is described and includes the default and allowed values that you can use.
+
+Compiling the source code
+-------------------------
+
+When following the official documentation to install the Wazuh manager :ref:`from sources <sources_installation>` (or the :ref:`Wazuh agent <agent-sources>`), the user runs the ``install.sh`` script. This will automatically compile the source code before installing it, but some customizations can be made prior to the script execution.
+
+To compile the code with ``make``, the user needs to change to the ``/src`` directory:
```

Comment:
I think is better to change the second sentence by _the working directory must be where the `MAKEFILE` resides, in this case the `src` directory of the `Wazuh` respository_ or something similar.

Reply:
Fixed!